view_controllers/OBARegionListViewController.m

```diff
@@ -343,7 +343,13 @@ - (void)showLocationServicesAlert {
         UIAlertView *view = [[UIAlertView alloc] init];
         view.title = NSLocalizedString(@"Location Services Disabled", @"view.title");
         view.message = NSLocalizedString(@"Location Services are disabled for this app. Some location-aware functionality will be missing.", @"view.message");
-        [view addButtonWithTitle:NSLocalizedString(@"Dismiss", @"view addButtonWithTitle")];
+        view.delegate = self;
+        [view addButtonWithTitle:NSLocalizedString(@"Okay", @"Ok button")];
+
+        if (&UIApplicationOpenSettingsURLString != NULL) {
```

Comment:
Per http://stackoverflow.com/a/25453598 this should properly test if the device can open settings. I can't install the iOS 7 simulator so I can't verify 100%.